todo_app/tests/test_client.py

```diff
@@ -1,27 +1,56 @@
 import pytest
+import requests
+
+from lxml import html
+from todo_app.trello.trello_base import TrelloBase
+from todo_app.trello.trello_list import TrelloList
+from todo_app.trello.trello_item import TrelloItem
+from todo_app.trello.trello_api import TrelloAPI
+
+from datetime import datetime, timedelta
 from dotenv import load_dotenv, find_dotenv
 from todo_app.app import create_app
 
 @pytest.fixture
 def client():
-    file_path = find_dotenv('.env.test.')
+    file_path = find_dotenv('.env.test')
     load_dotenv(file_path, override=True)
 
     test_app = create_app()
 
     with test_app.test_client() as client:
         yield client
 
-#@pytest.fixture
-#def mock_get_request():
-
-
-#def test_index_page(mock_get_request, client):
-#    response = client.get('/')
-
-
-#def test_index_page(client):
-#    response = client.get('/')
-#    assert response != None
-
+def test_index_page(monkeypatch, client):
+
+    def mock_get_all_trello_items():
+        TrelloBase.todo_list_id = 1
+        TrelloBase.done_list_id = 2
+        TrelloBase.doing_list_id = 3
+
+        trello_list = TrelloList()  
```

Comment:
We could share some of this setup with the view_model tests - it's quite overwhelming in this class having so much of it being setup!